lib/poda_payload.py

```diff
@@ -132,10 +168,17 @@ def get_data(self, vh):
             obj = self.s3.Object(self.bucketname, vh).get()
         except botocore.exceptions.ClientError as e:
             if e.response['Error']['Code'] == "404" or e.response['Error']['Code'] == 'NoSuchKey':
-                printdbg("Data does not exist for vh: %s" % vh)
-                return ''
+                tagData = self.storage_provider.getTagged(vh)
+                # The object does exist.
+                if (tagData.get("data") is None):
+                    printdbg("Data does not exist for vh: %s" % vh)
+                    raise
```

Comment:
change in functionality return '' to raise, is this intended? what are the consequences?

Reply:
@sidhujag, the Original code snippet's logic also use `raise` when the file is not found. 
kindly refer to: https://github.com/syscoin/sentinel/blob/fe952557cc8e2067857c69bb194efb4660f50986/lib/poda_payload.py#LL139C1-L139C22

I am not sure if this was done intentionally we simply mimicked the current login with the proposed change that was requested